src/site/stages/build/process-cms-exports/schemas/output/node-regional_health_care_service_des.js

```diff
@@ -12,10 +12,11 @@ module.exports = {
       type: 'array',
       items: {
         entity: {
-          title: { type: 'string' },
+          title: { type: ['string', 'TBD'] },
```

Comment:
I don’t think TBDis a valid output type

Reply:
you can ignore this, we were testing something